scaffolding-chef-infra/tests/user-linux-api/tests/test.bats

```diff
@@ -10,6 +10,11 @@ SCAFFOLD_PKG_CHEF_CLIENT_VERSION="$(echo "${scaffold_chef_client}" | cut -d/ -f3
   [ "${result}" = "${scaffold_cacerts}" ]
 }
 
+@test "API: rubygems_url matches default.toml" {
+  result="$(grep '^rubygems_url = .*' /hab/svc/${TEST_PKG_NAME}/config/client-config.rb | cut -d ' ' -f3)"
+  [ "${result}" = "${cfg_rubygems_url}" ]
```

Comment:
I think for this to work it will need an to also be exported as a variable in the `test.sh` script.